commitlint/plugins.ts

```diff
@@ -262,6 +262,44 @@ export abstract class Plugins {
         ];
     }
 
+    public static defaultRevertMessage(
+        headerStr: string,
+        bodyStr: string | null,
+        when: string
+    ) {
+        let offence = false;
+        let isRevertCommitMessage = headerStr.toLowerCase().includes("revert");
+
+        const negated = when === "never";
+
+        if (isRevertCommitMessage) {
+            let isDefaultRevertHeader =
+                headerStr.match(/^[Rr]evert ".+"$/) !== null;
+
+            if (isDefaultRevertHeader) {
+                if (bodyStr !== null) {
+                    let lines = bodyStr.split("\n");
+                    offence =
+                        // 40 is the length of git commit hash.
```

Comment:
@realmarv please move this comment down, there's no reason to not put the comment next to the line that is referring to

Reply:
Done